src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both to avoid potential margin collapse. Here, you have a right margin on `.stars__star`, which is fine, but ensure that you're consistent throughout the rest of your stylesheet.